lib/Config/Initializer.php

```diff
@@ -382,4 +389,12 @@ private function setupWoocommerceTransactionalEmails() {
       $this->wcTransactionalEmails->useTemplateForWoocommerceEmails();
     }
   }
+
+  private function setupWoocommerceBlocksIntegration() {
+    $wcEnabled = $this->wcHelper->isWooCommerceActive();
+    $wcBlocksEnabled = $this->wcHelper->isWooCommerceBlocksActive( '6.3.0-dev' );
```

Comment:
This version may need updating when versions are finalised.

Reply:
We don't write spaces in the parents, could you please remove them so that the code looks like this? 
```
$wcBlocksEnabled = $this->wcHelper->isWooCommerceBlocksActive('6.3.0-dev');
```